praw/models/reddit/modmail.py

```diff
@@ -59,10 +59,10 @@ def parse(cls, data, reddit, convert_objects=True):
             conversation[entity] = reddit._objector.objectify(
                 conversation[entity])
 
+        if data.get('user'):
```

Comment:
We need to be able to ignore the `user` key when it doesn't exist, both here and in the forthcoming method for creating a new conversation. Using `convert_objects` to accomplish this was a clumsy solution. Hence, this commit.